rotkehlchen/data_import/importers/nexo.py

```diff
@@ -179,34 +180,20 @@ def _consume_nexo(
             )
             self.add_history_events(write_cursor, [event])
         elif entry_type == 'Exchange':
-            input_asset = asset_from_nexo(csv_row['Input Currency'])
-            input_amount = deserialize_asset_amount_force_positive(csv_row['Input Amount'])
-            out_event = HistoryEvent(
-                event_identifier=(event_identifier := f'{NEXO_PREFIX}{hash_csv_row(csv_row)}'),
-                sequence_index=0,
-                timestamp=(timestamp_ms := ts_sec_to_ms(timestamp)),
-                location=Location.NEXO,
-                amount=input_amount,
-                event_type=HistoryEventType.TRADE,
-                event_subtype=HistoryEventSubType.SPEND,
-                asset=input_asset,
-                location_label=transaction,
-                notes=f'{entry_type} from Nexo',
+            self.add_history_events(
+                write_cursor=write_cursor,
+                history_events=create_swap_events(
+                    event_identifier=f'{NEXO_PREFIX}{hash_csv_row(csv_row)}',
+                    timestamp=ts_sec_to_ms(timestamp),
```

Comment:
same here it seems that you can convert once the timestamp to ms and reuse it in the different places